shared-src/net/wurstclient/features/mods/other/MurderMysteryMod.java

```diff
@@ -122,39 +123,46 @@ public void onUpdate()
 		updateMS();
 		if(!hasTimePassedM(250))
 			return;
-		whoHasSword();
+		if(amINOTTheMurderer())
+			whoHasSword();
 		updateLastMS();
 	}
 
-	private void whoHasSword()
+	private boolean amINOTTheMurderer()
```

Comment:
As a rule of thumb you should avoid negative names. That way it is easier to read in logical expressions.

As a simple example `! isNotTrue()` reads much worse than `! isTrue()`. The performance impact either way should be negligible.

This can be harder, but negative field/variable names should also be avoided when possible for the same reason.